utils.py

```diff
@@ -72,3 +74,68 @@ def get_raw_data(fpath_set):
     assert len(raw.annotations) == 1200, err_msg
 
     return raw
+
+
+def get_behavioral_data(fpath_set):
+    """Read behavioral data to DataFrame.
+
+    Does the following:
+        - Reads in behavioral data as csv
+        - Checks if data has the expeceted length
+
+    Parameters
+    ----------
+    fpath_set : string
+        Absolute path to the csv file containing metadata
+
+    Returns
+    -------
+    metadata : pd.DataFrame
+        A pandas DataFrame containing metadata in columns and trials in rows.
+    """
+    #
+    metadata = read_csv(fpath_set)
+
+    # Set some known metadata
+    # Sanity check we have the expected number of events
+    err_msg = f"    >>> {len(metadata)} != the expected 1200"
+    assert len(metadata) == 1200, err_msg
+
+    return metadata
```

Comment:
I think I would kill this entire function --> it's basically just "read_csv" with a small sanity check, ... I think these two lines would be fine to write out. Especially because in future script, the sanity check is not needed anymore, because the data won't change, and the `00` script is always run first in the pipeline

Reply:
does it hurt to leave it in now?

Reply:
doesn't hurt, but it's a very quick adjustment that looks nicer down the line.